sound/smooth_swing_config.h

```diff
@@ -13,6 +13,7 @@ class SmoothSwingConfigFile : public ConfigFile {
     CONFIG_VARIABLE(Transition2Degrees, 160.0f);
     CONFIG_VARIABLE(MaxSwingVolume, 3.0f);
     CONFIG_VARIABLE(AccentSwingSpeedThreshold, 0.0f);
+    CONFIG_VARIABLE(AccentSlashAccelerationThreshold, 0.0f);
```

Comment:
Should we default this to on?

Reply:
If we do, we should default accents to on too, no?

Reply:
no
I mean that the can default to on IF accent swings are on.

Reply:
ok, set default value to 260, so if accent swing threshold is set in smoothsw.ini, slashes will default to on as well, provided there are slsh files present.